batchflow/models/tf/unet.py

```diff
@@ -59,14 +59,35 @@ class UNet(EncoderDecoder):
     def default_config(cls):
         config = super().default_config()
 
-        config['initial_block'] = dict(layout='cna cna', kernel_size=3, filters=64)
+        config['initial_block'] = dict(layout='cna cna', kernel_size=3)
 
         config['body/encoder/num_stages'] = 4
-        config['body/encoder/blocks'] = dict(layout='cna cna', kernel_size=3, filters=[128, 256, 512, 1024])
+        config['body/encoder/order'] = ['downsampling', 'block']
+        config['body/encoder/blocks'].update(dict(layout='cna cna', kernel_size=3))
```

Comment:
Why `update`, not `assign`? It should work just fine (I double checked)

Reply:
assignment works similarly as update indeed, but this is really misleading

made assignment here for now, but created issue https://github.com/analysiscenter/batchflow/issues/417 after discussion with @roman-kh